Explain that Go does not include occurrences in check definitions or built-in filters (#2313)
sensu/sensu-docs: content/sensu-go/5.19/installation/upgrade.md

Transport handlers are not supported by Sensu Go, but you can create similar functionality with a pipe handler that connects to a message bus and injects event data into a queue.

### Filters
Sensu Go includes three new built-in [event filters][9]: only-incidents, only-metrics, and allow-silencing.
Sensu Go does not include a built-in check dependencies filter or a filter-when feature.

Ruby eval logic from Sensu Core 1.x is replaced with JavaScript expressions in Sensu Go, opening up powerful ways to filter events based on occurrences and other event attributes.
As a result, **Sensu Go does not include the built-in occurrence-based event filter in Sensu Core 1.x**, which allowed you to control the number of duplicate events that reached the handler.
You can replicate the occurrence-based filter's functionality with Sensu Go's [repeated events filter definition][10].

> #### Fatigue check filter asset

Reviewer: ```suggestion
#### Fatigue check filter
```

> In Sensu Go, the [fatigue check filter asset][11] uses annotations to tune the number of occurrences and interval for alerts on a per-check or per-entity basis and control whether resolution events will pass through.

Reviewer: ```suggestion
For Sensu Go users, we recommend the [fatigue check filter][11], a JavaScript implementation of the `occurrences` filter from Sensu 1.x.
This filter looks for [check and entity annotations][33] in each event it receives and uses the values of those annotations to configure the filter's behavior on a per-event basis.
```

> The [Sensu Translator version 1.1.0][18] retrieves occurrence and refresh values from a Sensu Core 1.x check definition and outputs them as annotation values in a Sensu Go check definition.

Reviewer: ```suggestion
The [Sensu Translator version 1.1.0][18] retrieves occurrence and refresh values from a Sensu Core 1.x check definition and outputs them as annotations in a Sensu Go check definition, compatible with the fatigue check filter.
```

> The [fatigue check filter asset][11] uses these [annotations][33] to manage alert fatigue.

Reviewer: ```suggestion
```

However, the Sensu Translator doesn't automatically add the fatigue check filter asset or the filter configuration you need to run it.

> To use the fatigue check filter asset, you must [register it][15] and create a correctly configured [event filter definition][19].

Reviewer: I think there's a third step here: add the event filter to the list of filters on applicable handlers

> To use the fatigue check filter asset, you must [register it][15] and create a correctly configured [event filter definition][19].

Author: ```suggestion
To use the fatigue check filter asset, you must [register it][15], create a correctly configured [event filter definition][19], and [add the event filter][34] to the list of filters on applicable handlers.
```

[30]: /images/web-ui-entity-warning.png
[31]: https://sensu.io/contact?subject=contact-sales/
[32]: https://blog.sensu.io/one-year-of-sensu-go

> [33]: https://github.com/nixwiz/sensu-go-fatigue-check-filter/#configuration-1

Author: ```suggestion
[33]: https://github.com/nixwiz/sensu-go-fatigue-check-filter/#configuration-1
[34]: ../../guides/reduce-alert-fatigue/#assign-the-event-filter-to-a-handler-1
```
